src/modules/hub.bicep

```diff
@@ -79,6 +87,60 @@ module dataFactory 'Microsoft.DataFactory/factories/deploy.bicep' = {
   }
 }
 
+resource storageAccountLookup 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
+  name: storageAccount.name
+}
+
+resource blobService 'Microsoft.Storage/storageAccounts/blobServices@2021-06-01' = {
```

Comment:
Thinking out loud: Since we talked about breaking apart from CARML, should we move this into the storageAccounts/deploy.bicep file or maybe move all that into a storage.bicep file directly in this folder? I'm fine with any of the 3 approaches. Just thinking out loud... When I first saw this, it felt a little like creating tech debt that we'll need to clean up later. That said, I don't want to bloat the PR, so we could also do that in a separate PR. Thoughts? @ankur-ms? @tsilvers-ms?

Reply:
Will leave as-is for now